experiment_prototype/scan_classes/sequences.py

```diff
@@ -304,8 +303,23 @@ def __init__(self, seqn_keys, sequence_slice_dict, sequence_interface, options):
         self.seqtime = 2*self.options.tr_window_time + self.pulses[-1]['pulse_timing_us'] + \
                        self.last_pulse_len
 
-        # TODO change numberofreceivesamples to oversample so that no sample dropping
-        # has to happen due to edge effects (account for filter length).
+        blanks = []
+        sample_time = 1.0/float(self.options.output_sample_rate)
+        first_sample_time = sample_time/2.0  # TODO this needs to be calculated appropriately
```

Comment:
wrong! placeholder. check this

Reply:
need start time, decimation rate, sample rate and initialization time to calculate properly

Reply:
this will take me more time - I will create an issue for this. #78